mobile/src/utils/sort.js

```diff
@@ -0,0 +1,43 @@
+const dirMap = {
+  // greater-than
+  gt: { asc: 1, desc: -1 },
+  // less-than
+  lt: { asc: -1, desc: 1 },
+};
+
+const doSort = (A, B, property, direction = 'ASC') => {
+  const a = A[property];
+  const b = B[property];
+
+  if (a < b) {
+    return dirMap.lt[direction.toLowerCase()];
+  }
+  if (a > b) {
+    return dirMap.gt[direction.toLowerCase()];
+  }
+  return 0;
+};
+
+export const createSorter = (...args) => {
+  return (A, B) => {
+    let ret = 0;
+
+    args.some((sorter) => {
+      const { property, direction = 1 } = sorter;
+      const value = doSort(A, B, property, direction);
+
+      if (value === 0) {
+        // they are equal, continue to next sorter if any
+        return false;
+      }
+      // they are different, stop at current sorter
+      ret = value;
+
+      return true;
+    });
+
+    return ret;
+  };
+};
+
+export default { createSorter };
```

Comment:
Is this required? You already have named export on createSorter

Reply:
There is an eslint rule that enforces us to use default exports: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

Reply:
Ok, then no need for named export and export default without destructuring should be enough (if we have only one function exported)